include/realizations/catchment/Formulation_Manager.hpp

```diff
@@ -271,7 +277,9 @@ namespace realization {
             }
 
             /**
-             * @brief Get the formatted output root
+             * @brief Get the formatted output root: check the existence of the output_root directory defined
+             * in realization. If true, return the directory name. Otherwise, throw a runtime error and 
+             * request the user to create that directory
```

Comment:
```suggestion
             * @brief Get the formatted output root: check the existence of the output_root directory defined
             * in realization. If true, return the directory name. Otherwise, try to create the directory
             * or throw an error on failure.
```

The documentation is slightly out of date from a previous commit

Reply:
Note: doesn't need to be exactly these changes, I wrote this with respect to my comment on `mkdir`

Reply:
Yeah. I for got about it. Good catch.